src/CentrifugeRouter.sol

```diff
@@ -10,15 +10,18 @@ import {ICentrifugeRouter} from "src/interfaces/ICentrifugeRouter.sol";
 import {IPoolManager} from "src/interfaces/IPoolManager.sol";
 import {IEscrow} from "src/interfaces/IEscrow.sol";
 import {IGateway} from "src/interfaces/gateway/IGateway.sol";
+import {TransientStorage} from "src/libraries/TransientStorage.sol";
 
 contract CentrifugeRouter is Auth, ICentrifugeRouter {
+    using TransientStorage for bytes32;
+
+    // The slot holding the initiator state, transiently. bytes32(uint256(keccak256("initiator")) - 1)
+    bytes32 public constant INITIATOR_SLOT = 0x390f14ca25a428cfdaf9fa3f25a94d06d247d3fa36c5a914ca10e13a4120db3c;
```

Comment:
why having a comment and hardcoded value when you can have 
` bytes32 public constant INITIATOR_SLOT = bytes32(uint256(keccak256("initiator")) - 1)`
Constants are evaluated during compile time.

Reply:
True, fixed!